library/src/main/java/com/firebase/ui/com/firebasei/ui/authimpl/FirebaseOAuthToken.java

```diff
@@ -0,0 +1,29 @@
+package com.firebase.ui.com.firebasei.ui.authimpl;
+
+/**
+ * Created by abehaskins on 11/3/15.
+ */
+public class FirebaseOAuthToken {
+    public String token;
+    public String secret;
+    public String uid;
+    public String provider;
+    public int mode;
+
+    public static final int SIMPLE = 1;
```

Comment:
Please use a proper `enum` for this: https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html